### PR TITLE
[Resource Sharing Form] fixed alignment of the input fields in the form 

### PR DESCRIPTION
**Pull request by shivamgaur99**
## Issue that this pull request solves
 Closes: #935

fixed alignment of input text field of resource sharing form below are screenshots

## Types of changes
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist
_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
image
image

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Screenshot 2024-05-22 at 9 47 47 PM

it still breaking for `ipad mini` and `ipad air` screens, please fix the same

**Comment by shivamgaur99** [comment]
@Kajol-Kumari issue fixed, checked in all devices now its not breaking

image
image

File: backend/.env [hunk 1]
```diff
@@ -1,9 +1,9 @@
-MONGO_DB_URL=<mongoDB_URL>
```
Comment (Kajol-Kumari) [inline backend/.env:1]: cane we please revert this file change?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
please revert `.env` change it will be good to be merged.

File: backend/.env (hunk 1 above)
Comment (shivamgaur99) [reply backend/.env:1]: yeah sure

**Pull request merged**